src/components/Search.js

```diff
@@ -409,7 +409,10 @@ class Search extends Component {
                                     downshiftProps.highlightedIndex
                                 }
                                 loading={loading}
-                                parsedSuggestions={rawSuggestions}
+                                parsedSuggestions={rawSuggestions.filter(
+                                    suggestion =>
+                                        suggestion._type !== 'collections',
```

Comment:
Since we're now using es6 apps so you the `_type` will be always the same `_doc`, instead you can use the `type` property of source.